docs/genai/howto/build-from-source.md

```diff
@@ -87,12 +87,18 @@ Build from source and copy the include and libraries into `ORT_HOME`
 On Windows
 
 ```cmd
-build.bat --build_shared_lib --skip_tests --parallel [--use_cuda] --config Release
+build.bat --build_shared_lib --skip_tests --parallel [--use_dml] [--use_cuda] --config Release
```

Comment:
Should we specify that only one of `--use_dml` and `--use_cuda` can be selected at the same time (I don't know the best way to show this here, maybe `[--use_dml | --use_cuda]`

Reply:
Good call